packages/frontend/src/components/accounts/RecoverAccountSeedPhrase.js

```diff
@@ -107,9 +107,13 @@ class RecoverAccountSeedPhrase extends Component {
                 await recoverAccountSeedPhrase(seedPhrase);
                 await refreshAccount();
             }, (e) => {
-                if (e.message.includes('Cannot find matching public key')) {
-                    this.setState({ showCouldNotFindAccountModal: true });
+
+                if (IMPORT_ZERO_BALANCE_ACCOUNT) {
+                    if (e.message.includes('Cannot find matching public key')) {
```

Comment:
should we use error codes in the future?

Reply:
Yep will address this in a follow-up PR for import zero balance with link/ledger